### PR TITLE
feat(@air/zephyr): update tooltip to have custom styles 

### PR DESCRIPTION
**Pull request by isaacwashere**

### CODE REVIEW

**Review by kylemh — APPROVED** [review]
Thanks, I hate it

File: packages/zephyr/src/Tooltip.tsx [hunk 1]
```diff
@@ -172,9 +194,16 @@ export const Tooltip = ({
...
           zIndex: baseZIndex + 1,
+          ...(containerStyles as any),
         }}
       >
-        <Text variant="text-ui-14" tx={{ color: 'currentColor' }}>
+        <Text
+          variant="text-ui-14"
+          tx={{
+            color: 'currentColor',
+            ...(boxContentStyles as any),
```
Comment (andyhqtran) [inline packages/zephyr/src/Tooltip.tsx:204]: This isn't a box

File: packages/zephyr/src/Tooltip.tsx [hunk 2]
```diff
@@ -130,10 +144,18 @@ export const Tooltip = ({
...
   sideOffset = 10,
+  tx = {},
   withBorder = true,
   withArrow = true,
   'data-testid': testID,
 }: TooltipProps) => {
+  const {
+    tooltipArrow: arrowStyles,
+    tooltipBorder: borderStyles,
+    tooltipContentBox: boxContentStyles,
```
Comment (andyhqtran) [inline packages/zephyr/src/Tooltip.tsx:155]: `tooltipArrow` -> `TooltipArrow`

Comment (andyhqtran) [reply packages/zephyr/src/Tooltip.tsx:155]: All of these will need to be updated

**Review by andyhqtran — CHANGES_REQUESTED** [review]

File: packages/zephyr/src/Tooltip.tsx (hunk 1 above)
Comment (isaacwashere) [reply packages/zephyr/src/Tooltip.tsx:204]: Gotcha, I can change it to `textContentStyles`

File: packages/zephyr/src/Tooltip.tsx (hunk 2 above)
Comment (isaacwashere) [reply packages/zephyr/src/Tooltip.tsx:155]: Gotcha 👍

**Review by andyhqtran — APPROVED** [review]

**Pull request merged**